test/unit/test_base64_stream.py

```diff
@@ -133,7 +133,7 @@ def build_test_cases():
     test_cases = []
 
     # exact single-shot, varying multiples
-    for size in (1, 2, 3, 4, 5, 6, 7, 222, 1024):
+    for size in (0, 1, 2, 3, 4, 5, 6, 7, 222, 1024):
```

Comment:
Per offline discussion, reminder to add negative (non-`-1`) value to `test_cases`.

Also: can these be combined to some degree with the values on 139-140?

Reply:
done